docusaurus.config.js

```diff
@@ -3,16 +3,18 @@ const { unsupportedVersions } = require("./src/versions");
 
 const latestVersion = require("./src/versions").versionMappings[0].docsVersion;
 
+const docsSiteUrl = process.env.DOCS_SITE_URL || "https://docs.camunda.io";
+
```

Comment:
Take advantage of this environment variable as much as possible (which is really only twice, but still).